src/sb_sw_lib_tests.c.h

```diff
@@ -879,6 +879,50 @@ _Bool sb_test_shared_secret_cavp_1(void)
     return 1;
 }
 
+_Bool sb_test_compressed_pub_shared_secret(void) {
```

Comment:
This isn't a bad test but it doesn't actually validate point decompression since the sign is irrelevant for ECDH. Can you find an ECDSA signature verification test that uses a compressed public key?